Source/Data/BPMFBase.txt

```diff
@@ -11430,6 +11430,7 @@
 糔 ㄒㄧㄡˇ xiu3 vu.3 big5
 秀 ㄒㄧㄡˋ xiu4 vu.4 big5
 繡 ㄒㄧㄡˋ xiu4 vu.4 big5
+綉 ㄒㄧㄡˋ xiu4 vu.4 big5
```

Comment:
I'd suggest put "綉" after "螑" to avoid breaking the order expected by Plain Bopomofo users.

The reason is this: "綉" does not exist in [bpmf.cin](https://raw.githubusercontent.com/openvanilla/openvanilla/master/DataTables/bpmf.cin), which was originally Big5 based. Non-Big5 characters were added to [bpmf-ext.cin](https://raw.githubusercontent.com/openvanilla/openvanilla/master/DataTables/bpmf-ext.cin).

Many Plain Bopomofo users have muscle memory for those characters, especially considering "繡" is followed by "鏽" and "袖", two often used characters. 

We generate `data-plain-bpmf.txt` based on this file, and so the order matters.